appinfo/info.xml

```diff
@@ -19,7 +19,7 @@ Features:
 - Import from various password managers (KeePass, LastPass, DashLane, ZOHO, Clipperz.is )
 For an demo of this app visit [https://demo.passman.cc](https://demo.passman.cc)
 ]]></description>
-    <version>2.1.5</version>
+    <version>2.2.0</version>
```

Comment:
I think that @brantje wanted to keep 2.0 for when sharing vaults was ready, can you confirm @brantje?

Reply:
@animalillo I thought this update would exceed the patch in your MAJOR.MINOR.PATCH versioning, but if you disagree with that, it'll be 2.1.6
just say a word